docs/manage/config-policies.rst

```diff
@@ -0,0 +1,263 @@
+resource managers, lower priority values indicate higher priority.
+
+**Example 1: Limiting GPU/CPU Slots and Priority for Experiments**
+
+The following example demonstrates how to set a maximum of 4 slots (GPUs or CPUs) and a priority
+limit of 15. This configuration applies to both Kubernetes and Agent resource managers, though the
+priority system behaves differently across the two:
+
+-  In Kubernetes, higher priority values indicate higher priority.
+-  In Agent resource managers, lower priority values indicate higher priority.
+
+.. code:: yaml
+
+   constraints:
+     resources:
+       max_slots: 4
+       priority_limit: 15
+
+In this example: - Users cannot set ``max_slots`` greater than 4. - Users cannot set a priority
+lower than 15 for Agent RMs or higher than 15 for Kubernetes RMs.
+
+**Example 2: Limiting Resources for Agent Resource Manager (RM)**
+
+This example limits the number of slots and sets a priority limit specifically for an Agent resource
+manager (RM). The ``priority_limit`` is set to ``15``, and ``max_slots`` is set to ``1``. This means
+a user cannot set a priority value lower than 15 and cannot set ``max_slots`` greater than 1.
+
+.. note::
+
+   Tasks such as NTSC (notebooks, tensorboards, shells, and commands) and resource managers (RMs)
+   have priority levels ranging from 1 to 99. The default cluster priority is typically 42. In Agent
+   RMs, a lower priority number means a higher priority, while in Kubernetes RMs, a higher priority
+   number means a higher priority.
+
+.. code:: yaml
+
+   constraints:
+      priority_limit: 15
+      resources:
+         max_slots: 1
+
+If a user tries to set a notebook priority to 10 when the limit is 15, the request will fail, and
+the system will display an error message.
+
+Limit Maximum GPU Usage per Experiment
+======================================
+
+The following configuration policy example limits GPU usage per experiment. This is configured in
+the Config Policies > Experiments tab.
+
+.. code:: yaml
+
+   constraints:
+     resources:
+       max_slots: 4
+
+Priority Limits
+===============
+
+You can set a different priority limit for experiments and tasks.
+
+The following policy is configured in the Config Policies > Experiments tab.
+
+.. code:: yaml
+
+   constraints:
+     resources:
+       priority_limit: <experiment limit>
+
+The following policy is configured in the Config Policies > Tasks tab.
+
+-  Tasks
+
+.. code:: yaml
+
+   constraints:
+     resources:
+       priority_limit: <task limit>
+
+Invariant Configs
+=================
+
+Invariant configs are applied to all workloads in a given scope (global or workspace) and merged
+with user-submitted configurations.
+
+**Set Default Priority for All Experiments**
+
+The following configuration policy example sets a default priority for all experiments:
+
+.. code:: yaml
+
+   invariant_config:
+     resources:
+       priority: 5
```

Comment:
This format is correct (just calling out that we do not need to change the formatting of `priority` for `invariant_config`.